cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -0,0 +1,154 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
+)
+
+var seccompAdvisorCmd = &cobra.Command{
+	Use:   "seccomp-advisor",
+	Short: "Generate seccomp policies based on recorded syscalls activity",
+}
+
+var seccompAdvisorStartCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Start to monitor the system calls",
+	RunE:   runSeccompAdvisorStart,
+	SilenceUsage: true,
+}
+
+var seccompAdvisorStopCmd = &cobra.Command{
+	Use:   "stop",
+	Short: "Stop monitoring and report the policies",
+	RunE:   runSeccompAdvisorStop,
+	SilenceUsage: true,
+}
+
+var seccompAdvisorListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List existing seccomp traces",
+	RunE:   runSeccompAdvisorList,
+	SilenceUsage: true,
+}
+
+var (
+	outputMode string
+	seccompProfileName string
+)
+
+func init() {
+	// Add generic information.
+	rootCmd.AddCommand(seccompAdvisorCmd)
+	utils.AddCommonFlags(seccompAdvisorCmd, &params)
+
+	seccompAdvisorCmd.AddCommand(seccompAdvisorStartCmd)
+	seccompAdvisorStartCmd.PersistentFlags().StringVarP(&outputMode, "output-mode", "m", "Status", "The trace output mode, possibles values are ExternalResource or Status (default).")
+	seccompAdvisorStartCmd.PersistentFlags().StringVar(&seccompProfileName, "seccomp-profile-name", "", "The seccomp profile which will be created when using ExternalResource as value for --output-mode.")
+
+	seccompAdvisorCmd.AddCommand(seccompAdvisorStopCmd)
+	seccompAdvisorCmd.AddCommand(seccompAdvisorListCmd)
+}
+
+// runSeccompAdvisorStart starts monitoring of syscalls for the given
+// parameters.
+func runSeccompAdvisorStart(cmd *cobra.Command, args []string) error {
+	if params.Podname == "" {
+		return errors.New("Usage: kubectl gadget seccompadvisor start -p podname")
+	}
+
+	config := &utils.TraceConfig{
+		GadgetName:      "seccomp",
+		Operation:       "start",
+		TraceOutputMode: outputMode,
+		TraceOutput:     seccompProfileName,
+		CommonFlags:     &params,
+	}
+
+	traceID, err := utils.CreateTrace(config)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("%s\n", traceID)
+
+	return nil
+}
+
+// runSeccompAdvisorStop reports an already running trace which ID was given
+// as parameter.
+func runSeccompAdvisorStop(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errors.New("Usage: kubectl gadget seccomp-advisor stop global-trace-id\n")
+	}
+
+	callback := func(results []gadgetv1alpha1.Trace) error {
+		for _, i := range results {
+			if i.Spec.OutputMode == "ExternalResource" {
+				fmt.Printf("Succesfully created seccomp profile %q!\n", i.Spec.Output)
+			} else {
+				fmt.Printf("%v\n", i.Status.Output)
+			}
+		}
+
+		return nil
+	}
```

Comment:
nit: I think the print for the ExternalResource should be outside the loop, otherwise, you will have the following:

```
$ ./kubectl-gadget seccomp-advisor start -p hello-python -n seccomp-demo --output-mode ExternalResource --seccomp-profile-name jose
zJZeOYsLQeXhL5PI
$ ./kubectl-gadget seccomp-advisor stop zJZeOYsLQeXhL5PI
Succesfully created seccomp profile "jose"!
Succesfully created seccomp profile "jose"!

... 1 print for each node in the cluster ....

Succesfully created seccomp profile "jose"!

```

What about this:
```suggestion
	callback := func(results []gadgetv1alpha1.Trace) error {
		for _, i := range results {
			if i.Spec.OutputMode == "ExternalResource" {
				fmt.Printf("Successfully created seccomp profile %q!\n", i.Spec.Output)

				// No need to print this for each node
				break
			}

			if i.Status.Output != "" {
				fmt.Printf("%v\n", i.Status.Output)
			}
		}

		return nil
	}
```